Models/MuZero_torch_trainer.py

```diff
@@ -70,12 +71,43 @@ def filter_grad(grad, sample):
         
         handles = []
 
-        for i, sample in enumerate(samples):
-            handle1 = self.global_agent.prediction_policy_network.output_heads[i][0].weight.\
-                register_hook(lambda grad: filter_grad(grad, sample))
-            handle2 = self.global_agent.prediction_policy_network.output_heads[i][0].bias.\
-                register_hook(lambda grad: filter_grad(grad, sample))
-            handles.extend([handle1, handle2])
+        # TODO(lobotuerk) Find a proper way to do this
```

Comment:
W-why not just use the for loop? Is there some benefit to splitting the policy heads like this instead of keeping them in an array?

Reply:
https://github.com/silverlight6/TFTMuZeroAgent/pull/55#discussion_r1165485231